packages/api-db-mongodb/src/db/peer.ts

```diff
@@ -79,19 +80,22 @@ export class MongoDBPeerAdapter implements DBPeerAdapter {
 
   async updatePeer(
     id: string,
-    {hostURL, name, slug, token}: UpdatePeerInput
+    {hostURL, name, slug, token, isDisabled}: UpdatePeerInput
   ): Promise<OptionalPeer> {
+    const toUpdate = {
+      modifiedAt: new Date()
+    } as any
```

Comment:
To narrow down the typings a bit more, we could use

```suggestion
    } as Record<string, unknown>
```